lambda-deployment-orca/lambda-deployment-orca.gradle

```diff
@@ -28,19 +28,20 @@ dependencies {
   compileOnly (group: 'com.netflix.spinnaker.orca', name: 'orca-api', version: "${orcaVersion}")
   compileOnly (group: 'com.netflix.spinnaker.orca', name: 'orca-clouddriver', version: "${orcaVersion}")
   compileOnly (group: 'com.netflix.spinnaker.orca', name: 'orca-core', version: "${orcaVersion}")
-  compileOnly ("org.projectlombok:lombok:1.18.12")
-  compileOnly (group: 'com.netflix.spinnaker.kork', name: 'kork-artifacts', version: "${korkVersion}")
-  compileOnly (group: 'com.netflix.spinnaker.kork', name: 'kork-plugins-spring-api', version: "${korkVersion}")
-  implementation("com.amazonaws:aws-java-sdk:1.11.877")
-  implementation("org.apache.commons:commons-lang3:3.8.1")
-  implementation("com.github.ben-manes.caffeine:guava:2.8.5")
-  implementation("com.netflix.awsobjectmapper:awsobjectmapper:1.11.877")
-  implementation("org.apache.httpcomponents:httpclient:4.5.9")
-  implementation("org.apache.httpcomponents:httpcore:4.4.11")
-  implementation("com.squareup.okhttp3:okhttp:4.2.2")
-  implementation("com.squareup.okhttp3:okhttp-apache:3.13.1")
-  annotationProcessor("org.pf4j:pf4j:3.2.0")
-  annotationProcessor ("org.projectlombok:lombok:1.18.12")
+  compileOnly platform("io.spinnaker.kork:kork-bom:$korkVersion")
+  compileOnly ("org.projectlombok:lombok")
+  compileOnly (group: 'io.spinnaker.kork', name: 'kork-artifacts')
+  compileOnly (group: 'io.spinnaker.kork', name: 'kork-plugins-spring-api')
+
+  implementation(enforcedPlatform("io.spinnaker.kork:kork-bom:$korkVersion"))
+  implementation("com.amazonaws:aws-java-sdk")
+  implementation("com.netflix.awsobjectmapper:awsobjectmapper")
+
+  annotationProcessor("org.pf4j:pf4j:${pf4jVersion}")
```

Comment:
I bet it's possible to remove pf4jVersion from here and gradle.properties since kork defines it [here](https://github.com/spinnaker/kork/blob/89fa2b249d5917c69f6ec9432dee45f14789cb04/spinnaker-dependencies/spinnaker-dependencies.gradle#L159).

Reply:
ok i will remove kotlinVersion as well since nothin uses it